### DISCUSSION TITLE
Two urls for the same view

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7070] [Q&A] [answered]
I have a situation, where there are two url's, `http://site.dev/guest` and `http://site.dev/vip`.
The view, blueprint and data are exactly the same, the only difference is that for the guest I want to hide a section on the page.

I know I can use `{{ current_uri }}` to check which url is being hit, **but I don't know how I can couple two urls to the exact same page**

### I tried

Adding a default route to web.php `Route::statamic('/guest', 'pages.show');`, it returns the correct view but all the blueprint data is empty.

**Comment by jackmcdade** [comment, marked as answer]
Are those urls managed by entries? If so, all you need to do is set  `template: pages.show` so they both use the same template and voila! 💥

**Reply by notflip** [reply]
How do you mean url's managed by entries? It's a collection called pages, in which I have this 1 page, that I would like to reach from those 2 url's, thanks for helping out!

**Reply by jackmcdade** [reply]
Oh you want two urls to go to one actual entry? Wouldn't that be duplicate content though?

**Reply by notflip** [reply]
Well I was hoping I only need the content once, and the 2 url's can just point to that same content, inside the content I can use an if statement to check the current_uri and show or hide data base on that.

**Reply by jackmcdade** [reply]
Statamic isn't designed to manage duplicate content, so maybe you could go with the route option and then just fetch the content from the existing entry, maybe something like this could work?

```
Route::statamic('/guest', 'pages.show', function() {
    return Statamic\Facades\Entry::find('123abcxyz');
});
```

**Reply by notflip** [reply]
Thanks @jackmcdade is that syntax correct, I'm getting `array_merge(): Argument #2 must be of type array, Closure given`.
It seems the third argument is data to be passed along

**Reply by notflip** [reply]
@jackmcdade I think what you mean is using a general laravel route? That seems to be working!
```
Route::get('/guest', function() {
    return Statamic\Facades\Entry::find('3a1b67e0');
});
```

**Reply by jackmcdade** [reply]
There you go! I didn't test, just went for the hail mary 🏈  😅
